src/OrderManager/V7/OrderAgent.php

```diff
@@ -524,7 +526,7 @@ public function cancelStartedOrderPayment(): AbstractOrder
     public function updatePayment(StatusInterface $status): static
     {
         //log this for documentation
-        Simple::log('update-payment', 'Update payment called with status: ' . print_r($status, true));
+        $this->logger->info('update-payment', 'Update payment called with status: ' . print_r($status, true));
```

Comment:
casually notice this, shouldn't be the first parameter the message? there are 2 string passed in there

Reply:
See also https://github.com/pimcore/pimcore/issues/14907

Reply:
will lbe fixed in phpstan PR 37